source/postprocess/visualization/artificial_viscosity.cc

```diff
@@ -28,6 +28,15 @@ namespace aspect
   {
     namespace VisualizationPostprocessors
     {
+      template <int dim>
+      ArtificialViscosity<dim>::
+      ArtificialViscosity ()
+        :
+        CellDataVectorCreator<dim>("m/s/s")
```

Comment:
Shouldnt this be the unit of thermal conductivity (`W / (m*K)`, or in your expression `kg * m / ( s^3 * K)`?